columnq/Cargo.toml

```diff
@@ -13,18 +13,18 @@ path = "src/lib.rs"
 [dependencies]
 # pulling arrow-schema manually to enable the serde feature.
 # TODO: add serde feature in datafusion to avoid this workaround
-arrow-schema = { version ="26", features = ["serde"] }
```

Comment:
@chitralverma we might need to set the arrow-schema version to 38 so it remains the same as the version pinned in datafusion to fix the build issue, see https://github.com/apache/arrow-datafusion/commit/fc5d67aa12ec11dc43f61f1381db1dd54dcef784#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L51